test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Screens/AboutScreen.cs

```diff
@@ -0,0 +1,175 @@
+        /// </summary>
+        /// <param name="vpnSession">VPN session.</param>
+        public AboutScreen(WindowsDriver<WindowsElement> vpnSession)
+        {
+            var aboutView = vpnSession.FindElementByClassName("AboutView");
+            this.backButton = aboutView.FindElementByName("Back");
+            this.title = aboutView.FindElementByName("About");
+            var textBlocks = aboutView.FindElementsByClassName("TextBlock");
+            this.fpnTitle = textBlocks[1];
+            this.fpnSubtitle = textBlocks[2];
+            this.releaseTitle = textBlocks[3];
+            this.releaseVersion = textBlocks[4];
+            this.termsOfService = aboutView.FindElementByName("Terms of Service");
+            this.termsOfServiceButton = aboutView.FindElementByName("Open Terms of Service link");
+            this.privacyPolicy = aboutView.FindElementByName("Privacy Policy");
+            this.privacyPolicyButton = aboutView.FindElementByName("Open Privacy Policy link");
+            this.debug = aboutView.FindElementByName("Debug");
+            this.debugButton = aboutView.FindElementByName("Open Debug link");
+            this.viewLog = aboutView.FindElementByName("View Log");
+            this.viewLogButton = aboutView.FindElementByName("Open View log link");
+        }
+
+        /// <summary>
+        /// Get title on Language screen.
+        /// </summary>
+        /// <returns>The title string.</returns>
+        public string GetTitle()
+        {
+            return this.title.Text;
+        }
+
+        /// <summary>
+        /// Get FPN title on about screen.
+        /// </summary>
+        /// <returns>The FPN title.</returns>
+        public string GetFPNTitle()
+        {
+            return this.fpnTitle.Text;
+        }
+
+        /// <summary>
+        /// Get FPN subtitle on about screen.
+        /// </summary>
+        /// <returns>The FPN subtitle.</returns>
+        public string GetFPNSubtitle()
+        {
+            return this.fpnSubtitle.Text;
+        }
+
+        /// <summary>
+        /// Get release title.
+        /// </summary>
+        /// <returns>The release title.</returns>
+        public string GetReleaseTitle()
+        {
+            return this.releaseTitle.Text;
+        }
+
+        /// <summary>
+        /// Get release version.
+        /// </summary>
+        /// <returns>The release version.</returns>
+        public string GetReleaseVersion()
+        {
+            return this.releaseVersion.Text;
+        }
+
+        /// <summary>
+        /// Get terms of service.
+        /// </summary>
+        /// <returns>Terms of service.</returns>
+        public string GetTermsOfService()
+        {
+            return this.termsOfService.Text;
+        }
+
+        /// <summary>
+        /// Get privacy policy.
+        /// </summary>
+        /// <returns>The privacy policy.</returns>
+        public string GetPrivacyPolicy()
```

Comment:
Could remove these and just have a `public string PrivacyPolicy { get; private set; }` at the top of the class.

Reply:
`this.PrivacyPolicy` is an windows element. Also what I am trying to do here is to get the text on the windows element.